wpimath/src/test/native/cpp/filter/DebouncerTest.cpp

```diff
@@ -2,47 +2,54 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
-#include "frc/Debouncer.h"  // NOLINT(build/include_order)
+#include <wpi/timestamp.h>
 
-#include "frc/simulation/SimHooks.h"
+#include "frc/filter/Debouncer.h"
 #include "gtest/gtest.h"
+#include "units/time.h"
 
-using namespace frc;
+static units::second_t now = 0_s;
 
 TEST(DebouncerTest, DebounceRising) {
```

Comment:
The right way to do the setup/teardown here is with a test fixture class and TEST_F.  This will also contain the "now" variable so it doesn't have to be a global.